README.md

```diff
@@ -148,54 +172,51 @@ The plugin can be configured through the following optional attributes:
 
 ```js
 {
-  name:'insert-variable',
-  options: {
-    publisher: 'http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b',
-    defaultEndpoint: 'https://dev.roadsigns.lblod.info/sparql',
-    variableTypes: [
-          'text',
-          'number',
-          'date',
-          'location',
-          'codelist',
+  publisher: 'http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b',
+  defaultEndpoint: 'https://dev.roadsigns.lblod.info/sparql',
```

Comment:
The insert-variable plugin examples seem to be outdated.

Reply:
No, it still works like that, the only change is that you have to pass the config to the card instead of the plugin now

Reply:
Yes but it no longer expects a `template` but rather a `constructor` property.

Reply:
Sorry didn't realize that had changed should be fixed now